plugins/entity-validation/README.md

```diff
@@ -0,0 +1,51 @@
+# entity-validation
+
+This plugin creates a new page in Backstage where the user can validate the entities.
```

Comment:
Can you include the image from the PR here? A picture really helps people understand what the plugin is

Reply:
Makes total sense. I just added that screenshot to the file :smiley:

Reply:
Thanks! Also, I should have mentioned this in my initial comments but for this to show up properly on NPM you'll need to add this to your `package.json`:

```json
"repository": {
    "type": "git",
    "url": "https://github.com/backstage/backstage",
    "directory": "plugins/entity-validation"
  },
```

Without that you'll get a broken image when viewing this on NPM

Reply:
Indeed. Added it, just forgot it

Reply:
Awesome, many thanks! 🚀